demo/index.ts

```diff
@@ -261,8 +261,19 @@ const strike: MarkSpec = {
   },
 };
 
+const docNodeSpec = {
+  doc: {
+    /*
+     * We must allow 2 text elements to exists side by side
+     * so that when we delete an element we can then join it.
+     */
```

Comment:
Not entirely sure I understand this. Is this just a general bug that's being fixed here, or is it related to the nested elements?

Also, the comment here mentions two text elements but the content field is talking in terms of blocks, just thought I would flag in case that's unintended.

Reply:
Whoops - I think I unintentionally committed that after getting the node spec from Composer's editor.

Definitely not relevant to the PR!